conf/sample/nse-memif.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: nse
+  name: nse
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app: nse
+  name: nse
+  namespace: default
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    app: nse
+  name: nse
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: nse
+subjects:
+  - kind: ServiceAccount
+    name: nse
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        networkservicemesh.io: "true"
+        networkservicemesh.io/app: "nse"
+    spec:
+      serviceAccount: nse
+      containers:
+        - name: nse-1
+          image: networkservicemesh/nse-memif
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: INIT_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          volumeMounts:
```

Comment:
We will need to get per-Pod directories injected into NSCs and NSEs rather than mounting the whole NSM directory to ensure proper isolation and security.